src/Microsoft.VisualStudio.ProjectSystem.Managed.TestServices/ProjectSystem/ProjectTreeParser/ProjectTreeParser.cs

```diff
@@ -266,12 +266,27 @@ private void ReadFields(MutableProjectTree tree)
                         ReadIcon(tree, expandedIcon: true);
                         break;
 
+                    case "DisplayOrder":
+                        tokenizer.Skip(TokenType.Colon);
+                        tokenizer.Skip(TokenType.WhiteSpace);
+                        ReadDisplayOrder(tree);
+                        break;
+
                     default:
                         throw _tokenizer.FormatException(ProjectTreeFormatError.UnrecognizedPropertyName, $"Expected 'FilePath', 'Icon' or 'ExpandedIcon', but encountered '{fieldName}'.");
                 }
             }
         }
 
+        private void ReadDisplayOrder(MutableProjectTree tree)
+        {
+            Tokenizer tokenizer = Tokenizer(Delimiters.PropertyValue);
+
+            var ident = tokenizer.ReadIdentifier(IdentifierParseOptions.None);
```

Comment:
>ident [](start = 16, length = 5)

can we just call this `identifier`? We tend to not abbreviate in this repo

Reply:
Sure.